aea/aea_builder.py

```diff
@@ -284,11 +292,36 @@ def set_max_reactions(self, max_reactions: Optional[int]) -> "AEABuilder":
 
         :param max_reactions: int
 
-        :return: None
+        :return: self
         """
         self._max_reactions = max_reactions
         return self
 
+    def set_decision_maker_handler(
+        self, decision_maker_handler_dotted_path: str, file_path: Path
+    ) -> "AEABuilder":
+        """
+        Set decision maker handler class.
+
+        :param decision_maker_handler_dotted_path: the dotted path to the decision maker handler
+        :param file_path: the file path to the file which contains the decision maker handler
+
+        :return: self
+        """
+        dotted_path, class_name = decision_maker_handler_dotted_path.split(":")
+        module = load_module(dotted_path, file_path)
+        try:
+            import pdb; pdb.set_trace()
+            _class = getattr(module, class_name)
+            self._decision_maker_handler_class = _class
```

Comment:
@marcofavorito not currently working. Try: `aea create my_agent && cd my_agent && aea scaffold decision-maker-handler && aea run`. What we want is a `NotImplementedError` to be raised as the class is instantiated..but we cannot locate the class..

Reply:
after the fix, it seems to work